docs/data/introduction/installation/installation.md

```diff
@@ -0,0 +1,187 @@
+# Installation
+
+<p class="description">Install the necessary packages to start building with MUI X components.</p>
+
+MUI X packages are available through the free MIT-licensed Community Plan, or the commercially-licensed Pro and Premium Plans.
+See [the licensing page](/x/introduction/licensing/) for complete details.
+
+Note that you only need to install the packages corresponding to the components you're using—e.g. Data Grid users don't need to install Date and Time Pickers.
+
+## Dependencies
+
+MUI X components have a peer dependency on `@mui/material`.
+To install Material UI, run:
+
+npm
```

Comment:
Make more explicit? Like in https://mui.com/material-ui/getting-started/installation/#svg-icons

```suggestion
With **npm**:
```

Reply:
That works for me! I don't think we've defined a consistent style for this situation yet. I'll go with this, and add a note in the Style guide about it.